src/main/kotlin/com/bitclave/node/repository/search/offer/OfferSearchCrudRepository.kt

```diff
@@ -20,18 +19,6 @@ interface OfferSearchCrudRepository : PagingAndSortingRepository<OfferSearch, Lo
 
     fun deleteAllByOwner(owner: String): List<Long>
 
-    @Modifying
```

Comment:
@krykcmz , where is the comparable code in service? I hope we are not replacing single SQL with multiple delete statements in code

Reply:
@markshw The service is here:
https://github.com/bitclave/base-node/blob/develop/src/main/kotlin/com/bitclave/node/services/v1/OfferSearchService.kt#L843

The deletion of offerSearches seems efficient
The deletion of offerInteractions also efficient

We are deleting them just like in the suggestion
https://stackoverflow.com/questions/1842446/hibernate-is-much-faster-when-executing-a-native-query
https://github.com/bitclave/base-node/blob/develop/src/main/kotlin/com/bitclave/node/repository/search/interaction/PostgresOfferInteractionRepositoryImpl.kt#L55

There isn't any delete statement in the code and JPA repositories are optimized.